manuscripts/report.py

```diff
@@ -605,20 +606,31 @@ def create_data_figs(self):
         logger.info("Data and figs done")
 
     @classmethod
-    def period_name(cls, pdate, interval='quarter', offset=None):
+    def build_period_name(cls, pdate, interval='quarter', offset=None, start_date=False):
+        """ If not start_date, pdate is the end of the period date """
         # Just supporting quarters right now
         name = pdate.strftime('%Y-%m-%d') + ' ' + interval
+        months_in_quarter = 3
+        end_quarter_month = None
 
         if interval == 'quarter':
             if offset:
                 # +31d offset format
                 offset_days = int(offset[1:-1])
                 pdate = pdate - timedelta(days=offset_days)
-            name = pdate.strftime('%Y')
-            # The month is the next month 2017-04-01 to the current quarter
-            month = int(pdate.strftime('%m')) - 1
-            quarter = int(round(int(month) / 3, 0))
-            name += "-Q" + str(quarter)
+            if not start_date:
+                # The month is the next month 2017-04-01 to the current quarter
```

Comment:
I'm sorry, but I don't understand this comment

Reply:
I will improve it!

Thank you  very much for the review. I will work on the tests and doc issues!